src/Sulu/Bundle/DeveloperBundle/DataFixtures/ORM/LoadContactData.php

```diff
@@ -0,0 +1,177 @@
+            $manager->persist($title);
+        }
+
+        for ($i = 0; $i < 10; $i++) {
+            $contact = $this->createContact($faker, $manager);
+        }
+
+        $manager->flush();
+    }
+
+    private function createContact($faker, $manager)
+    {
+        $phone = new Phone();
+        $phone->setPhone($faker->phoneNumber);
+        $phone->setPhoneType($this->phoneType);
+        $manager->persist($phone);
+        $email = new Email();
+        $email->setEmail($faker->email);
+        $email->setEmailType($this->emailType);
+        $manager->persist($email);
+        $fax = new Fax();
+        $fax->setFax($faker->phoneNumber);
+        $fax->setFaxType($this->faxType);
+        $manager->persist($fax);
+
+        $contact = new Contact();
+        $contact->setFirstName($faker->firstName);
+        $contact->setLastName($faker->lastName);
+        $contact->setPosition('CEO');
+        $contact->setCreated(new \DateTime());
+        $contact->setChanged(new \DateTime());
+        $contact->setSalutation("Hello");
+        $contact->setDisabled(0);
+        $contact->setTitle($this->titles[rand(0, 2)]);
+        $contact->addPhone($phone);
+        $contact->addEmail($email);
+        $contact->addFax($fax);
+        $manager->persist($contact);
+
+        for ($i = 0; $i<= rand(0, 5); $i++) {
+            $address = $this->createAddress($faker, $manager);
+
+            $contactAddress = new ContactAddress();
+            $contactAddress->setAddress($address);
+            $contactAddress->setContact($contact);
+            $contactAddress->setMain(true);
+            $manager->persist($contactAddress);
+            $contact->addContactAddresse($contactAddress);
+            $address->addContactAddresse($contactAddress);
+        }
+
+        for ($i = 0; $i <= rand(0, 20); $i++) {
+            $note = new Note();
+            $note->setValue($faker->realText);
+            $contact->addNote($note);
+            $manager->persist($note);
+        }
+
+        return $contact;
+    }
+
+    private function createAddress($faker, $manager)
+    {
+        $address = new Address();
+        $address->setStreet($faker->streetName);
+        $address->setNumber($number = $faker->buildingNumber);
+        $address->setZip($zip = $faker->postcode);
+        $address->setCity($city = $faker->city);
+        $address->setState($faker->county);
+        $address->setCountry($this->country1);
+        $address->setAddressType($this->addressType);
+        $address->setBillingAddress(true);
+        $address->setPrimaryAddress(true);
+        $address->setDeliveryAddress(false);
+        $address->setPostboxCity($city);
+        $address->setPostboxPostcode($zip);
+        $address->setPostboxNumber($number);
+        $address->setNote($faker->realText);
+        $manager->persist($address);
+
+        return $address;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getOrder()
+    {
+        return 100;
```

Comment:
Have to load after the other fixtures as they all hack the ID to 1 - the configuration references the ID directly .. :(